src/cmd/manage_nsfs.js

```diff
@@ -513,11 +524,12 @@ async function update_account(data) {
     // for validating against the schema we need an object, hence we parse it back to object
     nsfs_schema_utils.validate_account_schema(JSON.parse(encrypted_data));
     if (update_name) {
-        await native_fs_utils.create_config_file(fs_context, accounts_dir_path, new_account_config_path, encrypted_data);
-        await native_fs_utils.delete_config_file(fs_context, accounts_dir_path, cur_account_config_path);
-    } else if (update_access_key) {
-        await native_fs_utils.update_config_file(fs_context, accounts_dir_path, cur_account_config_path, encrypted_data);
-    }
+        //write_stdout_response(ManageCLIResponse.AccountUpdated, "update_name 2 = " + update_name);
+        await nb_native().fs.unlink(fs_context, cur_root_account_config_path);
+        await nb_native().fs.symlink(fs_context, new_account_relative_config_path, new_root_account_config_path);
+    }// else if (update_access_key) {
```

Comment:
Why the "else if (update_access_key)" in a comment?

Reply:
By mistake, thanks.